spectrumany.py

```diff
@@ -86,15 +89,15 @@ def spectrumany(expression, color_list, selection='(all)', minimum=None, maximum
         val_range = maximum - minimum
         cmd.color(colors[0], selection)
 
-    steps = 60 / parts
+    steps = int(60 / parts)
```

Comment:
Could also use `//` here, like in https://github.com/speleo3/pymol-psico/blob/92667ea0c7e6265c9f588112a576eb3eeb61334a/psico/viewing.py#L182